qute.ls/com.redhat.qute.ls/src/main/resources/com/redhat/qute/project/datamodel/qute-resolvers.json

```diff
@@ -25,6 +25,23 @@
 				"{#for r in recordsList.take(3)}"
 			],
 			"url": "https://quarkus.io/guides/qute-reference#collections"
+		},
+		{
+			"signature": "or(self: T, arg : java.lang.Object) : boolean",
+			"description": "Returns the default value if the base object is null, empty or not found and the base Object otherwise",

```

Comment:
Please use the same description than reference guide:

`Outputs the default value if the previous part cannot be resolved or resolves to null.`